keyboards/model01/config.h

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2018 James Laird-Wah
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_H
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

Reply:
Additionally, you need to add `#include "config_common.h"` here, as well.

Reply:
Perhaps the recommended use of `#pragma once` could be added to the coding guidelines `docs/contributing.md`? I found the guidelines very helpful to know what you would prefer.